ui/ui-components/components/export/List.tsx

```diff
@@ -52,14 +55,55 @@ export default function ExportsList(props: Props) {
     }
   }
 
+  const retryExport = useCallback(
+    async (_export: Models.ExportType, index: number) => {
+      setLoading(true);
+      try {
+        const { count } = await client.requestAction<ExportsRetryFailed>(
+          "get",
+          `/exports/retryFailed`,
+          {
+            destinationId: _export.destination.id,
+            startTimestamp: _export.createdAt,
+            endTimestamp: _export.createdAt,
```

Comment:
Hmm, this _probably_ finds just the export you want, but it might grab others if they have the same `createdAt` which would be an unexpected consequence to the user. I think a safer way would be to modify the action to accept an individual `exportId`.